gen/perunServicesInit.pm

```diff
@@ -192,4 +195,12 @@ sub createFacilityNameFile($) {
 	close FACILITY_NAME_FILE;
 }
 
+sub setStandardJsonType {
+	$JSON_TYPE='json';
```

Comment:
I'm not happy with a function that changes a global variable. That is a bad practice.

Historically, the perunServicesInit module uses global variables to read data from a gen script. That's also not nice, but it's somehow acceptable for such a small, one-purpose program.

Therefore, please use a global variable as well or rewrite this whole module using the object oriented approach. So usage would be like following.
```perl
my $psi = perunServicesInit->new();
$psi->setJsonType(Perun::Agent::$JSON_SIMPLE)
```

Reply:
Ok, so we will use global variable, because changing the whole process to be object oriented will be pain at this moment.